analysis_options.yaml

```diff
@@ -10,29 +11,69 @@ linter:
   rules:
     - annotate_overrides
     - avoid_empty_else
+    - avoid_function_literals_in_foreach_calls
     - avoid_init_to_null
+    - avoid_null_checks_in_equality_operators
+    - avoid_relative_lib_imports
     - avoid_return_types_on_setters
+    - avoid_returning_null
+    - avoid_types_as_parameter_names
+    - avoid_unused_constructor_parameters
     - await_only_futures
     - camel_case_types
+    - cancel_subscriptions
     - comment_references
+    #- constant_identifier_names
```

Comment:
Perhaps add a link to https://github.com/dart-lang/logging/issues/43?

Reply:
ack